google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -440,9 +450,23 @@ public static class Builder {
     private boolean autoDetectDialect = false;
     private Duration waitForMinSessions = Duration.ZERO;
     private Duration acquireSessionTimeout = Duration.ofSeconds(60);
+    private Position releaseToPosition = getReleaseToPositionFromSystemProperty();
 
     private Clock poolMaintainerClock;
 
+    private static Position getReleaseToPositionFromSystemProperty() {
+      // NOTE: This System property is a beta feature. Support for it can be removed in the future.
+      String key = "SESSION_POOL_RELEASE_TO_POSITION";
```

Comment:
Maybe namespace this as `com.google.cloud.spanner.session_pool_release_to_position`?

Reply:
Yes, that is more idiomatic for System properties. All caps is for environment variables.